### PR TITLE
🎨  App design and Web presence alignment

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?

- Bring in colors from AppMotion
- Update login page to align with AppMotion design
- ...

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by pcrespov — COMMENTED** [review]
:tada:

**Review by sanderegg — APPROVED** [review]
nice!

**Review by elisabettai — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/ui/message/FlashMessage.js [hunk 1]
```diff
@@ -33,11 +33,23 @@ qx.Class.define("osparc.ui.message.FlashMessage", {
     this.base(arguments);
     this._setLayout(new qx.ui.layout.HBox(15));
 
+    const decorator = new qx.ui.decoration.Decorator().set({
+      width: 1,
+      style: "solid",
+      color: this.self().LOG_LEVEL_COLOR_MAP[level].color
+    });
```
Comment (ignapas) [inline services/static-webserver/client/source/class/osparc/ui/message/FlashMessage.js:40]: You can use `services/static-webserver/client/source/class/osparc/theme/Decoration.js` to add decorations

**Review by ignapas — CHANGES_REQUESTED** [review]
Awesome! just some comments:
* Window title bar and content background should be the same
image

**Comment by ignapas** [comment]
I merged latest `master` (pull before commit to avoid extra merge commits)

**Review by ignapas — COMMENTED** [review]
lgtm

**Pull request merged**